nano/node/transport/message_deserializer.hpp

```diff
@@ -90,6 +88,7 @@ namespace transport
 		nano::network_filter & publish_filter_m;
 		nano::block_uniquer & block_uniquer_m;
 		nano::vote_uniquer & vote_uniquer_m;
+		channel_read_fn_type channel_read_fn;
```

Comment:
The only thing I would change is name this better, it has nothing to do with channel, imo a short and suiting name is just `read_query_t read_query;`